dashdotdb/schemas/swagger.yaml

```diff
@@ -95,6 +68,56 @@ paths:
       responses:
         "200":
           description: successful operation
+        "404":
+          description: token not found
+
+  # Deployment Validation Operator
+  /deploymentvalidation/newtoken:
+    get:
+      operationId: dashdotdb.controllers.deploymentvalidation.newtoken
+      summary: Create a token for data upload
+      responses:
+        "200":
+          description: token created
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  token:
+                    type: string
+                    description: A unique token identifier
+        "500":
+          description: token creation error
+  /deploymentvalidation/closetoken:
+    post:
+      operationId: dashdotdb.controllers.deploymentvalidation.closetoken
+      summary: Close a token so it can not receive data
+      parameters:
+      - name: token
+        in: query
+        required: true
+        schema:
+          type: string
+      responses:
+        "200":
+          description: token closed
+        "404":
```

Comment:
pfff, I have doubts here... this is usually used when the url is wrong...

Reply:
I'm open to sane error return codes for the api calls.  It picked some, but maybe the 5xx range is better?

Reply:
no, no, 4xx is what we need here. Looking for inspiration in https://en.wikipedia.org/wiki/List_of_HTTP_status_codes#4xx_client_errors but found none. `404` should be fine